Project.toml

```diff
@@ -7,6 +7,7 @@ version = "0.12.1-DEV"
 AbstractAlgebra = "c3fe647b-3220-5bb0-a1ea-a7954cac585d"
 AlgebraicSolving = "66b61cbe-0446-4d5d-9090-1ff510639f9d"
 DocStringExtensions = "ffbed154-4ef7-542d-bbb7-c09d3a79fcae"
+Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
```

Comment:
Why is this needed? `Documenter` is declared a docs-dependency in `docs/Project.toml`.